__init__.py

```diff
@@ -1,15 +1,40 @@
 import asyncio
 
-from fastapi import APIRouter
+from fastapi import APIRouter, Request, Response
+from fastapi.routing import APIRoute
 
 from lnbits.db import Database
 from lnbits.helpers import template_renderer
 from lnbits.tasks import catch_everything_and_restart
+from typing import Callable
+from fastapi.responses import JSONResponse
 
 db = Database("ext_tpos")
 
+
+class LNURLErrorResponseHandler(APIRoute):
```

Comment:
this would handle all the /tpos API calls like LNURLErrorResponses right?

btw. i saw that handleer already multiple times in extensions, would it make sense to offer it from core in `helpers.py` or `lnurl.py`?